src/backends/llvm-backend.ts

```
@@ -95,122 +120,291 @@
+      
+      const resultReg = gen.allocRegister();
+      nodeResults.set(nodeId, resultReg);
+      
+      switch (node.type) {
+        case 'input':
+          gen.emit(`  ${resultReg} = bitcast i8* %input to i8*`);
+          break;
+          
+        case 'filter':
+          gen.emit(`  ; Filter node: ${nodeId}`);
+          const inputReg = node.inputs[0] ? nodeResults.get(node.inputs[0]) : '%input';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)